CONTRIBUTING.md

```diff
@@ -147,6 +147,11 @@ fix: Add missing static class variable to IconButton and Avatar
 2.  Run `yarn build`
 3.  Run `yarn start` to start [Storybook](https://storybook.js.org/)
 4.  Visit [http://localhost:9001/](http://localhost:9001/)
+5.  To quickly rebuild on file changes, run `yarn watch` in another terminal. This will rebuild
```

Comment:
👍

Reply:
This is only specific to tsc compilation for dist folders right? Step 3 already runs a storybook which can also be interpreted as "quickly rebuilds on file changes".

Should this be a separate statement? I don't know if I would run both `yarn start` and `yarn watch` at the same time really.

Reply:
@lychyi It is a subtle issue that doesn't always show up when developing.

Webpack (used by storybook) does not recompile everything. I'll lay out a scenario:

You're tasked with adding accessibility to the `Modal` component and you need add a `aria-labelledby` to the `Popup` component and an `id` to the `Header` component in the `Card` component and you need `Card` to accept a new prop for that to work.

You open up the `Popup.tsx` file and add `aria-labelledby` and save. You notice Storybook hot-reloads, but the Modal still doesn't show the attribute. Wha? You load up the Popup story and see that the `aria-labelledby` attribute shows up on the Popup example. Why?

You are working on the `Modal` storybook story which imports `@workday/canvas-kit-react-popup`. This module is actually coming from the `modules/popup/react/dist/es6/lib` folder and not `modules/popup/react/lib` folder. You only see changes in Storybook that don't go through an `import '@workday/canvas-kit-react-*'` dependency. In order to see that change show up, you'll need to run `yarn build` again which takes 120s. So this common type of development will need a `yarn watch` to get those types of changes faster (about 5s on my machine for the slowest module to update)